src/routes/pages/OrderCheckPage/OrderCheckPage.tsx

```diff
@@ -31,24 +68,60 @@ function OrderCheck() {
         </HeaderButton>
       </Header>
       <ContainerLayout>
-        <div css={OrderCheckContainerStyle}>
-          <Text varient="title">{TEXT.ORDER_CHECK}</Text>
-          <div>
-            <Text varient="caption">
-              총 {selectedCartData.length}종류의 상품 {totalProductQuantity}개를
-              주문합니다.
-            </Text>
-            <Text varient="caption">{TEXT.TOTAL_PRICE_CHECK_MESSAGE}</Text>
-          </div>
-          <div css={orderPriceContainerStyle}>
-            <Text varient="body">{TEXT.TOTAL_PRICE}</Text>
-            <Text varient="title">{totalPrice.toLocaleString()}원</Text>
-          </div>
-        </div>
+        <CartListTitle
+          title={TEXT.ORDER_CHECK}
+          description={`총 ${cart.typeCount}종류의 상품 ${cart.totalCount}개를 주문합니다.\n최종 결제 금액을 확인해주세요.`}
+        />
+        <ul css={OrderCheckCartListStyle}>
+          {cart.selectedCartItems.map((item: CartItemProps) => (
+            <OrderCartItem
+              key={item.id}
+              quantity={item.quantity}
+              product={{
+                imageUrl: item.product.imageUrl || Default,
+                name: item.product.name,
+                price: item.product.price,
+              }}
```

Comment:
```suggestion
              imageUrl={item.product.imageUrl || Default}
              name={item.product.name}
              price={item.product.price}
```
product를 넘겨주는 대신에, 이렇게 값을 넘겨주면 어떨까요!?